src/components/RichTextEditor/RichTextEditor.cy.tsx

```diff
@@ -150,6 +156,13 @@ describe('RichTextEditor Component', () => {
         cy.get(RICH_TEXT_EDITOR).should('contain.text', TEXT);
     });
 
+    it('should be possible to pass and change value', () => {
```

Comment:
The description of this test case seems missleading, as the actual test is just passing and not changing the value.

Reply:
Updated